IFC4x3/Properties/s/StorageCoefficient_36wybY1$X62RvbVDalUFnE/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="StorageCoefficient_36wybY1X62RvbVDalUFnE" Name="StorageCoefficient" UniqueId="c6ebc962-07f8-4609-be65-7cd92f78fc4e" PrimaryDataType="IfcReal" />
```

Comment:
Propose one of the ratio measures.

Reply:
Yes, it should be. Changed to IfcPositiveRatioMeasure in commit [d533080](https://github.com/bSI-InfraRoom/IFC-Specification/pull/626/commits/d533080c39808834c51d8c2557d8771fa1c508f3).